client/jetpack-onboarding/steps/site-type.jsx

```diff
@@ -20,6 +20,8 @@ import { saveJetpackOnboardingSettings } from 'state/jetpack-onboarding/actions'
 
 class JetpackOnboardingSiteTypeStep extends React.PureComponent {
 	handleSiteTypeSelection = siteType => () => {
+		this.props.recordJpoEvent( 'calypso_jpo_' + siteType + '_clicked' );
```

Comment:
I have a doubt regarding whether to name it like this, or have it save additional properties.

See https://github.com/Automattic/wp-calypso/pull/21578#discussion_r162004455 for more details.

Reply:
As per discussions and changes, this is modified accordingly here and in https://github.com/Automattic/wp-calypso/pull/21578.